charts/trino/templates/service.yaml

```diff
@@ -14,6 +14,12 @@ spec:
       targetPort: http
       protocol: TCP
       name: http
+    {{- if .Values.server.config.https.port }}
```

Comment:
This should be using a value from the `.Values.service` rather than `.Values.server.config.https.port`.

We need to figure out how to define more ports in the service section.

Reply:
Thanks for the review @bitsondatadev ! 

Looking at the `README.md` file,  there are currently three ways to set a port via Values:
* `service.port` defaults to `8080`
* `server.config.http` defaults to `8080`
* `server.config.https` defaults to `8443`

For this PR to work we need to be able to specify a HTTPS port, separate from HTTP port.

Based on how the Values is currently structured, we can't use `service.port` here, however I'm happy to do a PR to change the structure of the Values file if you could help me to understand what you'd like to see it structured as?  

Otherwise would appreciate if we could merge this PR now as it is working and follows the current Values file structure as documented.

Reply:
@bitsondatadev any thoughts on my previous comment? :)